hugo.toml

```diff
@@ -17,11 +17,12 @@ filename = "sitemap.xml"
 text_color = ""
 author = ""
 favicon = ""
-site_logo = ""
+site_logo = "/images/logo/T4EQ_Logo_w.png"
 description = "The last theme you'll ever need. Maybe."
 body_classes = "avenir bg-near-white"
 fontWeight = "normal"
 # choose a background color from any on this page: https://tachyons.io/docs/themes/skins/ and preface it with "bg-"
 # background_color_class = "bg-dark-blue "
 recent_posts_number = 1
-heading_font = "serif"
\ No newline at end of file
+heading_font = "serif"
+custom_css = ["css/custom.css"]
```

Comment:
I don't think this is getting applied properly, I get this in my browser console:
```
GET http://localhost:1313/css/custom.css net::ERR_ABORTED 404 (Not Found)
```

Which I think makes sense, because assets are meant to be queried with the [Go text template](https://pkg.go.dev/text/template) functionality as [`resources.Get`](https://gohugo.io/functions/resources/get/). 

To make sure this gets properly included, you can instead put it under `static/css/custom.css`, which will copy it to the output directory.

To be honest though, the page looks pretty nice without the extra custom CSS, so might not actually be needed (I couldn't find to what HTML tags the css would apply)